### PR TITLE
Readme tags

### LINKED ISSUE #54: Adding tags to Readme.md file
## Problem faced
The Readme file still lacks some information. The short information will let the user / contributors get a quick glance of the projec.

## Describe the solution you'd like
Adding shield.io tags will help the Readme File to me more specific for users

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
**Issue Number**

fixes #54

**Changes  made**

I have added tags for issues, forks, stars, license, twitter and languages

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
@Abhishek-kumar09 Please review this PR

File: README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@
...
 
+<br/>
+
+![Issues tag](https://img.shields.io/github/issues/codeforcauseorg/edu-client)
+![Forks tag](https://img.shields.io/github/forks/codeforcauseorg/edu-client)
+![Stars](https://img.shields.io/github/stars/codeforcauseorg/edu-client?style=social)
+![Licence](https://img.shields.io/github/license/codeforcauseorg/edu-client)
+![twitter](https://img.shields.io/twitter/follow/codeforcauseIn?style=social)
+![Languages](https://img.shields.io/github/languages/count/codeforcauseorg/edu-client)
+
```
Comment (Abhishek-kumar09) [inline README.md:17]: Align it to center and remove language shield

Comment (Abhishek-kumar09) [reply README.md:17]: Also add discord badge

Comment (kaiwalyakoparkar) [reply README.md]: Ok

Comment (kaiwalyakoparkar) [reply README.md]: For discord, the server admin has to enable widget settings on this server. Could you please do that

**Review by Abhishek-kumar09 — APPROVED** [review]

**Pull request merged**